src/nuclio/common/components/key-value-input/key-value-input.component.js

```diff
@@ -61,6 +62,7 @@
          * Initialization method
          */
         function onInit() {
+            ctrl.keyPlaceholder = lodash.defaultTo(ctrl.keyPlaceholder, 'Type key...');
```

Comment:
Hmm…
Seems like we could do this here:
```js
lodash.defaults(ctrl, {
    allowSelection: false,
    dropdownOverlap: false,
    keyOptional: false,
    keyPlaceholder: 'Type key...',
    onlyValueInput: false,
    submitOnFly: false,
    useAdditionalValue: false,
    valuePlaceholder: 'Type value...',
});
```
Please review the values I chose to make sure they make sense as the defaults.